features/projects.feature

```diff
@@ -118,6 +118,29 @@ Feature: Create and maintain projects
     And I should see a link to "Title New" on github
     And I should see a link to "Title New" on Pivotal Tracker
 
+  Scenario: Saving a new project with new PivotalTracker link format: success
```

Comment:
we've completely replicated a scenario here with only a single minor difference  It might be DRYer to use a scenario outline here:

https://github.com/cucumber/cucumber/wiki/Scenario-Outlines